.github/workflows/testing.yml

```diff
@@ -118,13 +120,20 @@ jobs:
     runs-on: ubuntu-latest
     timeout-minutes: 15
     services:
-      redis:
+      redis-insecure:
         image: redis
         # Hard coded port because environment variables not currently
         # supported for use outside of 'steps'
         ports:
           - 6379:6379
 
+      redis-secure:
+        image: redis
+          # Hard coded port because environment variables not currently
+          # supported for use outside of 'steps'
+        ports:
+          - 6378:6379
```

Comment:
is this mapping correct?

Reply:
Yeah. For testing Ive setup a secure and insecure redis instance and test them both. Have to make one port different so they dont interfere